v19.1/managed-orchestrate-an-app-using-kubernetes.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Orchestrate an application with Kubernetes
+summary: Learn how to build an application with Managed CockroachDB and orchestrate it using Kubernetes.
+toc: true
+build_for: [managed]
+---
+
+On top of Managed CockroachDB's built-in automation, you can use a third-party [orchestration](orchestration.html) system to simplify and automate even more of your operations, from deployment to scaling to overall cluster management. This page walks you through a simple demonstration, using the open-source [Kubernetes](http://kubernetes.io/) orchestration system.
+
+## Before your begin
+
+1. Follow Docker's [documentation](https://docs.docker.com/v17.12/docker-for-mac/install/) to install Docker.
+2. Follow Kubernetes' [documentation](https://kubernetes.io/docs/tasks/tools/install-minikube/) to install `minikube`, the tool used to run Kubernetes locally, for your OS. This includes installing a hypervisor and `kubectl`, the command-line tool used to managed Kubernetes from your local workstation.
+
+    {{site.data.alerts.callout_info}}Make sure you install <code>minikube</code> version 0.21.0 or later. Earlier versions do not include a Kubernetes server that supports the <code>maxUnavailability</code> field and <code>PodDisruptionBudget</code> resource type used in the CockroachDB StatefulSet configuration.{{site.data.alerts.end}}
+
+3. Build the sample Python application:
+
+    This tutorial uses the [sample To-Do app](https://github.com/cockroachdb/examples-python/tree/master/flask-sqlalchemy) application with Managed CockroachDB using [SQLAlchemy](https://docs.sqlalchemy.org/en/latest/). Refer to the [Build a Python app with Managed CockroachDB](managed-build-a-python-app.html) tutorial to build the sample application.
+
+## Step 1: Start a local Kubernetes cluster
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ minikube start
```

Comment:
I'm not sure `minikube` is that interesting to the managed audience. I'd expect this doc to be mainly about a managed k8s service like GKE or amazon's equivalent.